cli/src/command/wallet.rs

```diff
@@ -12,14 +12,27 @@ use log::LevelFilter;
 
 use crate::{error::Error, helper::get_password, println_log_info};
 
+const DEFAULT_NODE_URL: &str = "https://api.testnet.shimmer.network";
+const DEFAULT_WALLET_DATABASE_PATH: &str = "./stardust-cli-wallet-db";
+const DEFAULT_STRONGHOLD_SNAPSHOT_PATH: &str = "./stardust-cli-wallet.stronghold";
+const DEFAULT_LOG_LEVEL: &str = "debug";
+
 #[derive(Debug, Clone, Parser)]
 #[command(author, version, about, long_about = None, propagate_version = true)]
 pub struct WalletCli {
+    /// Set the path to the wallet database.
+    #[arg(long, value_name = "PATH", env = "WALLET_DATABASE_PATH", default_value = DEFAULT_WALLET_DATABASE_PATH)]
+    pub wallet_db_path: String,
+    /// Set the path to the stronghold snapshot file.
+    #[arg(long, value_name = "PATH", env = "STRONGHOLD_SNAPSHOT_PATH", default_value = DEFAULT_STRONGHOLD_SNAPSHOT_PATH)]
+    pub stronghold_snapshot_path: String,
+    /// Set the account to enter.
+    pub account: Option<String>,
+    /// Set the log level.
+    #[arg(short, long, default_value = DEFAULT_LOG_LEVEL)]
+    pub log_level: LevelFilter,
```

Comment:
Should we use `RUST_LOG`?

Reply:
I was also thinking about that, but maybe that's too tough for regular users, dunno.